go/controller/api/v1alpha1/autogenmodelconfig_types.go

```diff
@@ -25,16 +25,73 @@ const (
 )
 
 // ModelProvider represents the model provider type
-// +kubebuilder:validation:Enum=Anthropic;OpenAI;AzureOpenAI;Ollama
+// +kubebuilder:validation:Enum=Anthropic;OpenAI;AzureOpenAI;Ollama;GeminiVertexAI;AnthropicVertexAI
 type ModelProvider string
 
 const (
-	Anthropic   ModelProvider = "Anthropic"
-	AzureOpenAI ModelProvider = "AzureOpenAI"
-	OpenAI      ModelProvider = "OpenAI"
-	Ollama      ModelProvider = "Ollama"
+	Anthropic         ModelProvider = "Anthropic"
+	AzureOpenAI       ModelProvider = "AzureOpenAI"
+	OpenAI            ModelProvider = "OpenAI"
+	Ollama            ModelProvider = "Ollama"
+	GeminiVertexAI    ModelProvider = "GeminiVertexAI"
+	AnthropicVertexAI ModelProvider = "AnthropicVertexAI"
 )
 
+type BaseVertexAIConfig struct {
+	// The project ID
+	// +optional
+	ProjectID string `json:"projectID,omitempty"`
```

Comment:
I think this should be required?

Reply:
yep, forgot to change it once I removed the API key support